content/documents/order-returns.mdx

```diff
@@ -121,7 +123,7 @@ Response expected by OrderCloud
 }
 ```
 **Notes:**
-- The endpoint the Integration Event will `POST` the payload to at the `CustomImplementationUrl` is  `/calculateorderreturn`
+- The endpoint the Integration Event will `POST` the payload to the `CustomImplementationUrl` is `/calculateorderreturn`
```

Comment:
this change doesn't make sense

you post the payload TO an endpoint AT the custom implementation url

In the future, I would prefer if you asked clarifying questions on things you don't understand rather than opening your own PR